### DISCUSSION TITLE
Dual mvme

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1525] [General]
I have not looked I o this very far but figured I would ask before I spent hours looking.   I was out of storage I. My original hard drive.  So I found a dual NVMe hat for my raspberry pi and added another NVMe.    I now have two sticks 128 and 256 of storage.  I got everything working on the pi. Both sticks show up as storage when connected to the pi.   When I started allsky back up it was only showing the 128 stick.  Any ideas, what setting need to change.

**Comment by aaronwmorris** [comment]
Can you show the output of `df`?

**Reply by barrettepp** [reply]
Okay. More questions.  Have not got the df info yet but figured more out.   
I realized my new ssd was visible but not formatted so I have it showing up as a drive now.   
On to other questions -  
what do you do with all your data.   Do you delete your photos and keep the Timelapse’s and ketograms ect…. Or how else do you keep from being overwhelmed with storage.  
What is involved in moving the image folder. Can I move it another mounted disk.  It says I need to rerun the setup after changing placement. What is involved in that   Thank you.

**Reply by aaronwmorris** [reply]
The images I usually just let expire.  The default expiration time for images is 30 days, but this can easily be increased if you want to keep less images.

If you move the images folder, you just need to change the Image Folder setting in the config on the Image tab.  After you change the config, the setup.sh script needs to be run in order to ensure the folder has the correct permissions and the web server config is updated with the new location.

**Reply by barrettepp** [reply]
Aaron
I do not think my pictures are expiring.   I am pretty sure they are all there and are what are taking up all my space.   I don’t have access to my device during the day and I feel like I have looked for it before but how do I set it to delete my photos after 30 days/ set them to expire.   Sorry for all the questions trying to get my camera back up and going thank you

**Reply by aaronwmorris** [reply]
indi-allsky initiates an expiration job at the end of night and end of day (twice a day).  Even at 30 days, a higher resolution camera can use all the space on a 128GB storage device.

There is also a command line script to expire images and videos if you wish to perform this manually.  `./misc/expire_images.py`
